Unable to compile with -Werror -Wconversion

We are hitting compile errors when building with `-Werror=conversion` since release 1.12.0
```
googletest/googlemock/src/gmock-internal-utils.cc:204:61: error: conversion from ‘int’ to ‘char’ may change value [-Werror=conversion]
googletest/googletest/src/gtest-printers.cc:318:16: error: conversion from ‘int’ to ‘char’ may change value [-Werror=conversion]

```
I will try to propose a fix soon.

--- a/googlemock/src/gmock-internal-utils.cc
+++ b/googlemock/src/gmock-internal-utils.cc
@@ -201,7 +201,7 @@ GTEST_API_ void IllegalDoDefault(const char* file, int line) {
 constexpr char UnBase64Impl(char c, const char* const base64, char carry) {
   return *base64 == 0   ? static_cast<char>(65)
          : *base64 == c ? carry
-                        : UnBase64Impl(c, base64 + 1, carry + 1);
+                        : UnBase64Impl(c, base64 + 1, static_cast<char>(carry + 1));
 }
 
 template <size_t... I>

--- a/googletest/src/gtest-printers.cc
+++ b/googletest/src/gtest-printers.cc
@@ -315,7 +315,7 @@ void PrintTo(__uint128_t v, ::std::ostream* os) {
     low = low / 10 + high_mod * 1844674407370955161 + carry / 10;
 
     char digit = static_cast<char>(carry % 10);
-    *--p = '0' + digit;
+    *--p = static_cast<char>('0' + digit);
   }
   *os << p;
 }

